llvm/test/Transforms/Attributor/remove_out_args.ll

```diff
@@ -0,0 +1,16 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -S -passes=attributor < %s | FileCheck %s
+
+
+
+define i1 @foo(ptr %dst) {
```

Comment:
I doesn't look like the argument to this function was removed in the updated function foo ? 
Can you clarify? Also can you add some more tests? Are there any special / edge cases?

Reply:
Yes, that is because the function was not called, I just added this tes as a start point